modules/navier_stokes/doc/content/modules/navier_stokes/hcgdgfe.md

```diff
@@ -0,0 +1,99 @@
+# Hybrid Continuous/Discontinuous Galerkin Finite Element Navier Stokes
+
+In [!citep](pandare2016hybrid) the authors describe a scheme in which the
+velocity is discretized using a reconstructed discontinuous Galerkin (DG) method
+while the pressure is discretized using continuous Galerkin (CG) finite
+elements. While MOOSE has the ability to perform rDG(P0P1), otherwise known as a
+second order accurate finite volume method (see [insfv.md]), it does not support
+arbitrarily high-order reconstructions. It does, on the other hand, support
+arbitrarily high-order discontinuous Galerkin bases. In that vein we have
+implemented and tested a hybrid CG-DG scheme. A nice advantage of the hybrid
+CG-DG scheme is that it is intrinsically LBB stable, allowing equal polynomial
+order bases for the velocity and pressure. Additionally, because the velocity is
+discretized with a DG scheme, the momentum advection flux can be naturally
+upwinded without a complex upwinding scheme like as Streamline-Upwind
+Petrov-Galerkin (SUPG) (see [cgfe.md]).
+
+In the following we run through an example of a hybrid CG-DG input, in which we
+are using the scheme to solve a lid driven cavity problem with a Reynolds number
+of 200.
+
+The first block is for the mesh. We create a simple square and add a nodeset
+which we will use to pin the pressure, removing its nullspace (the
+discretization only involves the gradient of pressure so without something like
+a pressure pin, the pressure is only defined up to a constant).
+
+!listing modules/navier_stokes/test/tests/finite_element/ins/cg-dg-hybrid/lid-driven/hybrid-cg-dg.i block=Mesh
+
+We next introduce the variables. `u` is the x-velocity component while `v` is
+the y-velocity component. Note that we use a `MONOMIAL` basis but we could just
+as well have used `L2_LAGRANGE` or `L2_HIERARCHIC`. Note that the default basis
+order is 1 or `FIRST`. Similarly the default `family` is `LAGRANGE` so the the
+`pressure` variable is implicitly `LAGRANGE`.
+
```

Comment:
is there any constraints or guidelines on the variable families between pressure and velocities?

Reply:
Oh, wow, I'd love to know this too.  There's no stabilization here, so there's got to be an LBB condition on the mixed elements, doesn't there?  Or is merely using DG stabilization enough?

Reply:
Added a bit more text about this